### PR TITLE
Engineering Physics Minor

### PR DESCRIPTION
**Pull request by issacli-0821**
Added PHY minor YAML

### CODE REVIEW

File: backend/minors/PHY.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- name: Five Additional Courses
+  max_counted: 1
+  min_needed: 1
+  pdfs_allowed: 1
+  explanation: |-
+    For B.S.E. students, these should be five PHY courses at the 200 level or higher,
+    including a course in quantum mechanics (PHY 208).  Of these, at least four must be
+    listed or cross-listed as physics (PHY) courses.  A fifth may be a course with strong
+    physics content from other departments such as AST or CHM, but must be approved in
+    advance by the program's committee.
```
Comment (ggchiriac) [inline backend/minors/PHY.yaml:55]: Add <br><br>

File: backend/minors/PHY.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  to become increasingly relevant in the 21st century. The Program in Engineering
+  Physics, which provides students with a fundamental knowledge of physics,
+  together with problem-solving skills and an understanding of engineering, is
+  designed to address the needs of students seeking innovative careers in today's
+  technological age. In addition, it allows students to keep their options open
+  between physical sciences and engineering. Following completion of the engineering
+  physics program, students typically enter careers in engineering,
+  applied science or applied physics through research, teaching or entrepreneurial
+  engineering. Past graduates have also pursued other careers as diverse as medicine,
+  business and law.
```
Comment (ggchiriac) [inline backend/minors/PHY.yaml:18]: Add " < br >< br > "

**Review by ggchiriac — CHANGES_REQUESTED** [review]
Add the brs there

File: backend/minors/PHY.yaml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  max_counted: 1
+  min_needed: 1
+  explanation: |-
+    <i>Editor's Note: Students do not enroll in any numbered course for this requirement, so hoagieplan does not check it.</i><br><br>
+
+    Close collaboration with faculty is expected. Students are required to complete,
+    with a grade of B- or better, at least one semester of independent work in an
+    appropriate area.  Physics students are encouraged to have a professor in
+    engineering serve as a reader on their senior thesis.
+
```
Comment (G-Marin) [inline backend/minors/PHY.yaml:97]: IW should have no_req since student's can't enroll in numbered course for it

Comment (issacli-0821) [reply backend/minors/PHY.yaml]: Good catch

**Review by ggchiriac — APPROVED** [review]
lgtm

**Pull request merged**